mxcube3/routes/Diffractometer.py

```diff
@@ -1,10 +1,18 @@
 from flask import Response, jsonify, request, session
 from mxcube3 import app as mxcube
-from mxcube3.routes import Utils
+from mxcube3.routes import Utils, signals
 
 import json
 import logging
 
+def init_signals():
+    """
+    Connect all the relevant hwobj signals with the corresponding
+    callback method.
+    """
+    diffractometer = mxcube.diffractometer
+    diffractometer.connect("minidiffPhaseChanged", signals.minidiff_phase_changed)
```

Comment:
We should probably avoid to use the name minidiff to be more generic, maybe just phaseChanged ?

Reply:
the signal minidiffPhaseChanged comes like that from the hwobj.
I will change the method name in signals

Reply:
I see, but you are the one that crated the signal so imagine that you can change that as well. So that someone that does have phaseChanged but not a minidiff still can use the code without having to add minidiff to their signal name.

Reply:
not exactly, it comes from the GenericDiffractometer.
Do you have/use this signal?

Reply:
no big deal. Changing signal name to listen